contrib_bots/lib/gmail.py

```diff
@@ -0,0 +1,163 @@
+#instructions to install googleapiclient can be found in this bot's readme file.
+from __future__ import print_function
+import base64
+import httplib2
+import logging
+import mimetypes
+import os
+import re
+
```

Comment:
Check the existence of '.google-credentials.json' file before this `from googleapiclient import discovery, errors` statement and exit program if you can't find it. The error message is very confusing otherwise. It seems that Google API checks for the existence of the file when it is imported.

>/srv/zulip-venv/local/lib/python2.7/site-packages/oauth2client/_helpers.py:255: UserWarning: Cannot access /home/vagrant/.google-credentials.json: No such file or directory
  warnings.warn(_MISSING_FILE_MESSAGE.format(filename))
usage: run.py [--auth_host_name AUTH_HOST_NAME] [--noauth_local_webserver]
              [--auth_host_port [AUTH_HOST_PORT [AUTH_HOST_PORT ...]]]
              [--logging_level {DEBUG,INFO,WARNING,ERROR,CRITICAL}]
run.py: error: unrecognized arguments: lib/gmail.py --config-file /home/vagrant/.zuliprc